snapshots/devmapper/pool_device.go

```diff
@@ -162,7 +162,20 @@ func (p *PoolDevice) transition(ctx context.Context, deviceName string, tryingSt
 		result = multierror.Append(result, uerr)
 	}
 
-	return result.ErrorOrNil()
+	return errorOrNil(result)
+}
+
+func errorOrNil(e *multierror.Error) error {
```

Comment:
`Unwrap`? https://github.com/hashicorp/go-multierror/blob/master/multierror.go#L65

Reply:
I didn't know that. Will update.

Reply:
That's brand-new and the version we are using doesn't have that yet. I will rename the function from errorOrNil to Unwrap.